src/sonic-pit/pit-sysdiag/src/fan_tc.py

```diff
@@ -0,0 +1,402 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+__version__ = 'v0.1.0'
+__author__  = "Pytool Li<admin@pytool.com>"
+import sys
+import time
+import traceback
+from test_case import TestCaseCommon
+from errcode import *
+from function import load_platform_util_module
+
+
+DEFAULT_FAN_CONTROL_INTERVAL = 5  # interval for fan control command
+
+class FANTC(TestCaseCommon):
+    __FANUTIL_MODULE_NAME = "fanutil"
+    __FANUTIL_CLASS_NAME = "FanUtil"
+    __SENSORUTIL_MODULE_NAME = "sensorutil"
+    __SENSORUTIL_CLASS_NAME = "SensorUtil"
+
+    def __init__(self, index, logger, platform_cfg_file, case_cfg_file=None):
+        MODULE_NAME = "fan_tc" # this param specified the case config dirictory
```

Comment:
Not sure the comment in this line is correct.

Reply:
This comment is misaligned. Removed.